server/scsynth/CMakeLists.txt

```diff
@@ -193,7 +193,7 @@ if (LIBSCSYNTH)
 	set_property(TARGET libscsynth PROPERTY SOVERSION   1)
 endif()
 
-if (FFTW3F_FOUND)
+if (FFTW3F_FOUND AND NOT APPLE)
 	target_include_directories(libscsynth PUBLIC ${FFTW3F_INCLUDE_DIR})
 	target_link_libraries(libscsynth ${FFTW3F_LIBRARY})
```

Comment:
This should be added in APPLE for linking to succed!!

Reply:
For scsynth we want to use vDSP with Apple by default, not FFTW. But I agree that it seems that now it's not possible to use FFTW if one wishes to do so...

[Supernova seems to have this declaration](https://github.com/supercollider/supercollider/blob/3037bf299bf21618bdf11fce53bf53427e84f13a/server/supernova/CMakeLists.txt#L140), am I seeing this right?

Reply:
I was confused with the linking error
```
 "_rffts", referenced from:

      scfft_dofft(scfft*) in libsupernova.a(SC_fftlib.o)

  "_riffts", referenced from:

      scfft_doifft(scfft*) in libsupernova.a(SC_fftlib.o)
```
which seems related to https://github.com/supercollider/supercollider/blob/develop/common/SC_fftlib.cpp#L412
meaning that SC_FFT_GREEN is defined (and it should not)

Reply:
Ah thanks for the pointers @sonoro1234 ! Very much appreciated. I'll look into this...